### PR TITLE
Fix low acc problem of tensorflow/conv

### PR DESCRIPTION
**Pull request by tomykaira**
The cause was too large Conv2D output channels.
Under SGD, the gradient is too small and the first Conv2D layer learned almost nothing.

This patch changes the # filters to the same value as the keras sample.
The new version marked 95% accuracy when tested on GPU (tf. 1.10.0).

### CODE REVIEW

**Comment by cympfh** [comment]
Great!
